lib/package.rb

```diff
@@ -69,6 +69,13 @@ def self.is_fake?
     @is_fake
   end
 
+  # Remove /usr/local/lib64/*.la to resolve any "can not find /usr/local/*.la" compile errors when building the packages (only needed for x86_64).
+  # https://gnunet.org/faq-la-files
+  # https://stackoverflow.com/questions/42963653/libquadmath-la-is-not-a-valid-libtool-archive-when-configuring-openmpi-with-g
+  def self.remove_la_files
+    system "rm -rf #{CREW_LIB_PREFIX}/*.la"
+  end
```

Comment:
This is not the right location for this logic.  I think you should be able to accomplish the same thing in `crew` instead.